packages/demo/examples/01-xy-chart/ResponsiveXYChart.jsx

```diff
@@ -37,6 +37,12 @@ export function renderTooltip({ datum, seriesKey, color }) {
         <strong style={{ color }}>y </strong>
         {yVal && yVal.toFixed ? yVal.toFixed(2) : yVal}
       </div>
+      {data && (
```

Comment:
`index` should already be passed as a key in the arguments object, can we use that instead of searching the full array each time?